packages/hub/services/data-integrity-checks/scheduled-payments.ts

```diff
@@ -152,12 +166,53 @@ export default class DataIntegrityChecksScheduledPayments {
       errorMessages
     );
 
+    await this.addCrankAndRelayerErrorMessages(errorMessages);
+
     return {
       name: 'scheduled-payments',
       status: errorMessages.length > 0 ? 'degraded' : 'operational',
       message: errorMessages.length > 0 ? errorMessages.join('; ') : null,
     };
   }
+
+  async addCrankAndRelayerErrorMessages(errorMessages: string[]) {
+    let networkNames = config.get('web3.schedulerNetworks') as string[];
+
+    for await (let networkName of networkNames) {
+      let relayerBalance = await this.getRelayerFunderBalance(networkName);
+      let crankBalance = await this.getCrankBalance(networkName);
+
+      let nativeTokenSymbol = getConstantByNetwork('nativeTokenSymbol', networkName);
+
+      if (relayerBalance.lte(LOW_RELAYER_FUNDS_THRESHOLD)) {
+        errorMessages.push(
+          `Relayer balance low on ${networkName}: ${ethers.utils.formatEther(relayerBalance)} ${nativeTokenSymbol}`
+        );
+      }
+
+      if (crankBalance.lte(LOW_CRANK_FUNDS_THRESHOLD)) {
+        errorMessages.push(
+          `Crank balance low on ${networkName}: ${ethers.utils.formatEther(crankBalance)} ${nativeTokenSymbol}`
+        );
+      }
+    }
+  }
+
+  async getRelayerFunderBalance(networkName: string): Promise<BigNumber> {
+    let relayerUrl = getConstantByNetwork('relayServiceURL', networkName);
+
+    let responseText = await (await fetch(`${relayerUrl}/v1/about`)).text();
+    let relayerFunderPublicKey = JSON.parse(responseText).settings.SAFE_FUNDER_PUBLIC_KEY;
```

Comment:
Note to self: SAFE_FUNDER_PUBLIC_KEY is wrong, SAFE_TX_SENDER_PUBLIC_KEY is correct